cdk/domino_cdk/provisioners/eks/lambda/cluster_post_creation_tasks.py

```diff
@@ -0,0 +1,56 @@
+import json
```

Comment:
Is it possible to unit test this code somehow? We've used [moto](https://github.com/spulec/moto) for AWS-mocking before.

Reply:
The value of this UT is low. This function is basically 4 calls with the static parameters or parameters copied from env.

Let's talk

Reply:
As discussed, our goal is to make a unit-test framework for this kind of code.

This particular lambda is simple and just passes parameters from env to API but we can use is a guinea pig.

We will do unit tests as separate JIRA https://dominodatalab.atlassian.net/browse/PLAT-2494